tests/lib/Persistence/Legacy/Content/Type/ContentTypeHandlerTest.php

```diff
@@ -48,6 +49,9 @@ class ContentTypeHandlerTest extends TestCase
      */
     protected $updateHandlerMock;
 
+    /** @var \Ibexa\Core\Persistence\Legacy\Content\Type\StorageDispatcherInterface|\PHPUnit\Framework\MockObject\MockObject */
```

Comment:
For future reference, apparently we should actually use:
```suggestion
    /** @var \Ibexa\Core\Persistence\Legacy\Content\Type\StorageDispatcherInterface&\PHPUnit\Framework\MockObject\MockObject */
```